libraries/ms-common/test/test_chip_id.c

```diff
@@ -0,0 +1,32 @@
+#include "chip_id.h"
+
+#include <inttypes.h>
+#include "delay.h"
+#include "interrupt.h"
+#include "log.h"
+#include "misc.h"
+#include "soft_timer.h"
+#include "test_helpers.h"
+#include "unity.h"
+
+ChipId chip_id_1 = { 0 };
+ChipId chip_id_2 = { 0 };
+
+void setup_test(void) {
+  interrupt_init();
+  soft_timer_init();
+}
+
+void teardown_test(void) {}
+
+// Verifies chip ID is consistant and prints chip ID for debugging
+void test_chip_id(void) {
+  const uint32_t delay_second_ms = 10;
+  chip_id_1 = chip_id_get();
+  delay_ms(delay_second_ms);
+  chip_id_2 = chip_id_get();
+  for (int i = 0; i < 2; i++) {
+    LOG_DEBUG("%" PRIu32 "\n", chip_id_1.id[i]);
```

Comment:
A couple requests just to help with determining what a board's chip ID is:
* the upper bound on the loop should be 3 to loop through all three elements
* could you use hex instead of decimal when printing? This just requires you to use `PRIx32` instead of `PRIu32`. Also put an `0x` in front of the `%` so it prints out with an `0x` in front please.

Also, nitpick: you should use uint8_t or another fixed width type instead of `int`, even though it doesn't really matter.